### PR TITLE
draft of a post on the path from junior to mid

### PR DESCRIPTION
**Pull request by dbgrandi**
Still early, but enough content for me to put it up here.

### CODE REVIEW

File: source/posts/growing_beyond_junior.markdown [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+title: Growing Beyond Junior
+date: 2016/2/21
+published: true
+---
+
+I've previously written about [what a junior developer should know](http://dbgrandi.github.io/minimum_viable_programmer/) and [what makes a great programmer](http://dbgrandi.github.io/good_great_10x/). Today, I want to talk about the path from being a junior developer to being a mid-level developer.
+
+## When you are looking for a job...
+
+Let’s start out by assuming that you are a junior developer without a job. The basic premise is that you need to practice programming...a lot. According to experts, you can't just do more tutorials and practice making table views. In order to progress you need to have organized practice with feedback and reflection. This is known as [deliberate practice](https://en.wikipedia.org/wiki/Practice_(learning_method)#Deliberate_practice). For a lot of people, the way to do this will be by getting a job as developer. This will allow you to focus on the craft of programming every day.
```
Comment (orta) [inline source/posts/growing_beyond_junior.markdown]: use the special ellipsis `practice programming...` -> `…`

File: source/posts/growing_beyond_junior.markdown [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+## Beyond practice... Projects, Peers, Passion, and Play
+
+The concept of practicing programming is something that we can explore a bit and expand upon. I was talking with [@lazerwalker](https://twitter.com/lazerwalker) recently about this post and he pointed me to an [paper by Mitchel Resnick](http://web.media.mit.edu/~mres/papers/constructionism-2014.pdf) that looks at the design of the Scratch programming language through the lens of Four P’s of Creative Learning: Projects, Peers, Passion, and Play.
```
Comment (orta) [inline source/posts/growing_beyond_junior.markdown]: > pointed me to -an- paper

Comment (orta) [inline source/posts/growing_beyond_junior.markdown]: link to scratch

**Comment by dbgrandi** [comment]
I can always edit more later, but I'd rather :ship: faster.

**Pull request merged**